proxy/rpc/np_gas_price.py

```diff
@@ -140,3 +168,122 @@ async def get_neon_gas_price(
     async def get_native_token_list(self) -> list[_RpcNativeTokenResp]:
         gas_price = await self._server.get_gas_price()
         return list(_RpcNativeTokenResp.from_raw(token) for token in gas_price.token_dict.values())
+
+    @NeonProxyApi.method(name="eth_maxPriorityFeePerGas")
+    async def get_max_priority_fee_per_gas(self, ctx: HttpRequestCtx) -> HexUIntField:
+        # Fetch the compute units price across last several blocks (as specified in the cfg).
+        num_blocks = self._cfg.priority_fee_num_blocks_to_average
+        assert num_blocks > 0
+        historical_priority_fees: list[list[int]] = await self._db.get_historical_priority_fees(num_blocks)
+        assert num_blocks == len(historical_priority_fees)
+
+        # Take the weighted average across fetched blocks.
+        average_weighted_max_priority_fee = 0
+        for idx, block_priority_fees in enumerate(historical_priority_fees):
+            # Sanity check to avoid division by zero.
+            average_block_priority_fee = 0
+            if len(block_priority_fees) != 0:
+                average_block_priority_fee = sum(block_priority_fees) / len(block_priority_fees)
+            # The most recent block goes with the biggest weight.
+            average_weighted_max_priority_fee += average_block_priority_fee * (num_blocks - idx)
+        average_block_priority_fee /= num_blocks * (num_blocks + 1) / 2
+
+        # Convert it into ethereum world by multiplying by suggested_gas_price
+        # N.B. prices in the block are stored in microlamports, so conversion to lamports takes place.
+        _, token_gas_price = await self.get_token_gas_price(ctx)
+        return int(token_gas_price.suggested_gas_price * average_block_priority_fee / 1_000_000)
+
+    @NeonProxyApi.method(name="eth_feeHistory")
+    async def get_fee_history(
+        self,
+        ctx: HttpRequestCtx,
+        num_blocks: HexUIntField,
+        block_tag: RpcBlockRequest,
+        priority_fee_percentiles: list[int] | None,
```

Comment:
Please rename to:
`priority_fee_percentile_list`

Reply:
done